pkg/didcomm/protocol/didexchange/states.go

```diff
@@ -687,7 +687,7 @@ func (ctx *context) getMyDIDDoc(pubDID string, routerConnections []string, servi
 
 	if newService {
 		switch newDID.Service[0].Type {
-		case didCommServiceType:
+		case didCommServiceType, "IndyAgent":
```

Comment:
is this a new DIDComm service type? There's many places where this needs to be updated and must follow logic of mediaType profile for DIDComm V1 + Interop Flag. It means the Interop flag must become part of AFGO's context.

I'm currently updating bdd tests with DID docs missing the MediaTypeProfiles() and KeyAgreement() contexts, the new Interop flag must be added on top of these changes as well.

It's not enough to change this here. There's at least oob invitations as well.

Reply:
> is this a new DIDComm service type?
It's an old service type, from AIP1, that we only need to recognize in limited cases, with the intent that it goes away once ACA-Py is sure to no longer emit it. Adding support everywhere would be counterproductive.